src/frontend/pages/home/home.js

```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+import { Footer } from "../../components/footer/footer";
+import { Navbar } from "../../components/navbar/nav";
```

Comment:
barrel imports can be used here for the component folder

Reply:
Yeah thanks will do it